services/scraper/search_keyword_test.go

```diff
@@ -0,0 +1,34 @@
+package scraper_test
+
+import (
+	"fmt"
+	"net/url"
+
+	"go-crawler-challenge/services/scraper"
+	. "go-crawler-challenge/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Scraper/SearchKeyword", func() {
+	BeforeEach(func() {
+		keyword := "keyword"
+		visitURL := fmt.Sprintf("https://www.google.com/search?q=%s&lr=lang_en", url.QueryEscape(keyword))
+		cassetteName := "scraper/success_valid_params"
+
+		RecordCassette(cassetteName, visitURL)
+	})
+
+	Describe("#Call", func() {
+		Context("given valid params", func() {
```

Comment:
@Lahphim Is it possible to add other cases with top ads, bottom ads, etc.?

Reply:
@carryall I'll do that in next PR about saving result to database, because I need to clean up result from that ticket also